lib/webvalve.rb

```diff
@@ -35,8 +35,22 @@ def config_paths
       @config_paths ||= Set.new
     end
 
+    def logger
+      if defined?(::Rails)
+        ::Rails.logger | default_logger
```

Comment:
is there a reason you chose to use `|` instead of `||` here?

Reply:
it's a mistake, solved it